stable/prometheus-pushgateway/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.9.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.0.0
```

Comment:
You started from an old version of the master. You need to update your fork to be aligned before starting a PR.
https://github.com/helm/charts/blob/master/stable/prometheus-pushgateway/Chart.yaml

Reply:
The test failed for this reason. https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/16040/pull-charts-e2e/1179945867478568964/#1:build-log.txt%3A188

Reply:
No. I was wrong. The PR remained unattended for too much time and then become old. The solution is the same. Redo the PR starting from an updated chart fork.

Reply:
Thank you